### ISSUE TITLE
make install failing

### ISSUE THREAD

**Issue by CameronGo** [codahale/sneaker#25]
I don't see any mistakes I've made in following the instructions.  I manually cloned the aws sdk repo (due to #24).  When I run the make install it tells me the package is not a go package or is not in GOPATH, though as far as I can tell both are not true.  See below:

```
root@a3dce834372b:~/gopath/sneaker# export GOPATH=/root/gopath
root@a3dce834372b:~/gopath/sneaker# make install
touch cmd/sneaker/version.go
/root/gopath/bin/govendor install  -ldflags "-X main.version '8573936' -X main.goVersion 'go version go1.6.2 linux/amd64' -X main.buildTime '2016-08-31T02:40:51Z'" +local
Error: Package "/root/gopath/sneaker" not a go package or not in GOPATH.
Makefile:16: recipe for target 'install' failed
make: *** [install] Error 2
root@a3dce834372b:~/gopath/sneaker# ll
total 180
drwxr-xr-x 5 root root  4096 Aug 30 19:25 ./
drwxr-xr-x 6 root root  4096 Aug 30 19:29 ../
drwxr-xr-x 8 root root  4096 Aug 31 02:40 .git/
-rw-r--r-- 1 root root     7 Aug 30 19:25 .gitignore
-rw-r--r-- 1 root root    81 Aug 30 19:25 .travis.yml
-rw-r--r-- 1 root root   595 Aug 30 19:25 LICENSE
-rw-r--r-- 1 root root   875 Aug 30 19:25 Makefile
-rw-r--r-- 1 root root 10535 Aug 30 19:25 README.md
-rw-r--r-- 1 root root 58585 Aug 30 19:25 architecture.png
drwxr-xr-x 3 root root  4096 Aug 30 19:25 cmd/
-rw-r--r-- 1 root root   794 Aug 30 19:25 download.go
-rw-r--r-- 1 root root  1520 Aug 30 19:25 download_test.go
-rw-r--r-- 1 root root  2997 Aug 30 19:25 envelope.go
-rw-r--r-- 1 root root   924 Aug 30 19:25 envelope_test.go
-rw-r--r-- 1 root root   719 Aug 30 19:25 fake_kms_test.go
-rw-r--r-- 1 root root  1215 Aug 30 19:25 fake_s3_test.go
-rw-r--r-- 1 root root  1263 Aug 30 19:25 list.go
-rw-r--r-- 1 root root  3659 Aug 30 19:25 list_test.go
-rw-r--r-- 1 root root  1032 Aug 30 19:25 pack.go
-rw-r--r-- 1 root root  2294 Aug 30 19:25 packaging_test.go
-rw-r--r-- 1 root root   348 Aug 30 19:25 rm.go
-rw-r--r-- 1 root root   611 Aug 30 19:25 rm_test.go
-rw-r--r-- 1 root root   664 Aug 30 19:25 rotate.go
-rw-r--r-- 1 root root  3202 Aug 30 19:25 rotate_test.go
-rw-r--r-- 1 root root  1554 Aug 30 19:25 sneaker.go
-rw-r--r-- 1 root root   485 Aug 30 19:25 unpack.go
-rw-r--r-- 1 root root   863 Aug 30 19:25 upload.go
-rw-r--r-- 1 root root  1998 Aug 30 19:25 upload_test.go
drwxr-xr-x 3 root root  4096 Aug 30 19:25 vendor/

**Comment by stuartrexking** [comment]
@CameronGo This might work for you https://github.com/codahale/sneaker/pull/26

**Comment by dstnbrkr** [comment]
Fixed by #26
